### PR TITLE
Add pymongo 3.2.2

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@behackett I am adding pymongo to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge pymongo). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/pymongo-feedstock (and the other packages as <package>-feedstock) shortly after this PR is finished and merged. If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!  Is there anyone else that you think would want to be pinged about maintaining this build?

**Comment by jakirkham** [comment]
Thanks for doing this @ericdill. Please feel free to add me here. 😄

Some day I will get my act together and get a `mongo` PR up here. I have something WIP locally.

**Comment by ericdill** [comment]
@jakirkham Happy to add you. Done.

**Comment by jakirkham** [comment]
Thanks. 😄

**Comment by behackett** [comment]
Neat! Thanks for the heads up. Let me know if you run into any problems.

**Comment by jakirkham** [comment]
Given this is compiling some code, could we please run the test suite?

File: recipes/pymongo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - bson
+    - gridfs
+    - pymongo
+
+about:
+  home: https://github.com/mongodb/mongo-python-driver
+  license: Apache Software License
```
Comment (jakirkham) [inline recipes/pymongo/meta.yaml]: Please specify license version (e.g. "Apache 2.0").

**Comment by ericdill** [comment]
> Given this is compiling some code, could we please run the test suite?

It's a little tricky because some of the pymongo test suite expects an active mongodb that it can connect to.  I can run the test suite with `python setup.py test` and get a reasonable success rate: 

```
Ran 326 tests in 216.931s

FAILED (errors=9, skipped=60)
```

I'm not sure that we want to try and stand up mongo databases in the test section of conda build.  `conda install mongodb` is an option for linux/mac, but such a conda package is not available for windows.  Thoughts?

**Comment by jakirkham** [comment]
> `conda install mongodb` is an option for linux/mac, but such a conda package is not available for windows.  Thoughts?

I would have figured we would make it a run-time dependency.

**Comment by ericdill** [comment]
@jakirkham Added a test script that will start a mongo server, run the test suite and then kill the mongo server after the test suite runs. This test script is successfully running the test suite on travis and circle

**Comment by behackett** [comment]
I just fixed a number of PyMongo tests that errored when MongoDB was not running. See [this commit](https://github.com/mongodb/mongo-python-driver/commit/1ee5e27f9a5b63648f9b09253520717dd1e9e367).

**Comment by ericdill** [comment]
@behackett Awesome. So in theory we would be able to run the test suite on windows too after the next tag?

**Comment by behackett** [comment]
I don't see why not. We run CI tests against Windows. We just rarely test without a server running. I discovered those test problems while developing big endian support for the C extensions on a Sparc box without MongoDB access.

**Comment by ericdill** [comment]
Thanks @pelson :tada:

**Pull request merged**